src/data.ts

```diff
@@ -139,7 +139,7 @@ export async function convertToIstanbulCoverage(
       }
 
       let relativePath;
-      if (url.protocol === 'webpack:') {
+      if (url.protocol.startsWith('webpack')) {
```

Comment:
The protocol in the Next.js compiled output is `webpack-internal:///`, but this line expects `webpack:///`.

Reply:
Makes sense. The change, I mean. All those minute differences in webpack setups are beyond me.